main.js

```diff
@@ -0,0 +1,47 @@
+const renderer = Renderer(posts)
+const logic = Logic()
+
+const removePosts = function () {
+	const post = $(this).closest('.container').attr('id')
+	console.log(renderer(logic.removePost(post)))
```

Comment:
In each of these next functions, you should do the `logic` invocation and the `renderer` separately one after the other. When you invoke `renderer` you should pass through `logic.getPosts()`.